capellambse_context_diagrams/collectors/generic.py

```diff
@@ -159,19 +150,25 @@ def exchange_data_collector(
             "targets": [render_adj.get("targets", target.uuid)],
         },
     )
+
+    label = collect_label(data.exchange)
+    for filter in data.filter_iterable:
+        try:
+            label = filters.FILTER_LABEL_ADJUSTERS[filter](
+                data.exchange, label
+            )
+        except KeyError:
+            logger.exception(
+                "There is no filter labelled: '%s' in "
+                "filters.FILTER_LABEL_ADJUSTERS",
+                filter,
+            )
```

Comment:
It can be reached when someone is adding an unknown key to the `filter_iterable`. This is exactly behaving like it should. The diagram is still rendered, neglecting the unknown filter key (This ultimately means that the user didn't register a filter function in the FILTER_LABEL_ADJUSTERS registry, maybe that is not clear? But if the user is looking into the docs he should know.) and informing the user about the exception.